app/controllers/actions/AffinityGroupIdentifierAction.scala

```diff
@@ -69,6 +71,37 @@ class AffinityGroupIdentifierAction[A] @Inject()(action: Action[A],
     }
   }
 
+  private def authoriseOrg(request : Request[A],
+                           enrolments : Enrolments,
+                           internalId : String,
+                           action: Action[A]
+                          ): Future[Result] = {
+
+    val enrolmentKey = "HMRC-TERS-ORG"
+    val identifier = "SAUTR"
+
+    val continueWithoutEnrolment =
+      action(IdentifierRequest(request, internalId, AffinityGroup.Organisation, enrolments))
+
+    enrolments.getEnrolment(enrolmentKey).fold(continueWithoutEnrolment){
+      enrolment =>
+        enrolment.getIdentifier(identifier).fold{
+          Logger.warn("missing SAUTR key for HMRC-TERS-ORG")
+          continueWithoutEnrolment
+        }{
+          enrolmentIdentifier =>
+            val utr = enrolmentIdentifier.value
+
+            if(utr.isEmpty) {
+              Logger.warn("UTR is empty")
```

Comment:
Can this not be a warning log level, every registration will now log this on every controller that is predicated with auth.